pkg/provisioner/ironic/ironic.go

```diff
@@ -321,6 +321,52 @@ func (p *ironicProvisioner) getNode() (*nodes.Node, error) {
 	}
 }
 
+// Verifies that node has port assigned by Ironic.
+func (p *ironicProvisioner) nodeHasAssignedPort(ironicNode *nodes.Node) bool {
+	opts := ports.ListOpts{
+		Fields: []string{"node_uuid"},
+	}
+
+	opts.NodeUUID = ironicNode.UUID
+
+	pager := ports.List(p.client, opts)
+	if pager.Err != nil {
+		p.log.Info("node has no assigned port", "error", pager.Err)
```

Comment:
Let's use the debugLog for these.

Reply:
rebased to use versioned logger with level 6

Reply:
Please just use `p.debugLog()`.

Reply:
Sorry, did not notice we have explicit debug level logger there, thought you were talking about versioned one, fixed.